src/actions/actionCreators.js

```diff
@@ -8,3 +8,44 @@ import * as types from './actionTypes';
   })
 
 */
+export const login = (username, password) => (dispatch) => {
+  // replace the url to login route
+  // console.log(username, password);
```

Comment:
Hey David, can you remove this comment for console log?

Reply:
done